### PR TITLE
Refine docs

### PR DESCRIPTION
**Pull request by anotherrachel**
### What problem does this PR solve? 

This PR refines Chaos Mesh docs after a global review.

### Check List 

Tests 

 - Unit test
 - E2E test
 - Manual test (add detailed scripts or steps below)
 - No code

Code changes

 - Has Go code change
 - Has CI related scripts change
 - Has Terraform scripts change

Side effects

 - Breaking backward compatibility

Related changes

 - Need to update the documentation

### Does this PR introduce a user-facing change?:

```release-note
NONE
```

### CODE REVIEW

File: doc/get_started_on_minikube.md [hunk 1]
```diff
@@ -1,58 +1,62 @@
...
-> - `netem chaos` is only supported for `minikube` clusters >= version 1.6.
->
->   In `minikube`, the default virtual machine driver's image doesn't contain the `sch_netem` kernel module in smaller versions. You can use `none` driver (if your host is Linux with the `sch_netem` kernel module loaded) to try these chaos actions on `minikube` or [build an image with sch_netem by yourself](https://minikube.sigs.k8s.io/docs/contributing/iso/).
+## Limitations
+
+There are some known restrictions for Chaos Operator deployed in the Minikube cluster:
+
+- `netem chaos` is only supported for Minikube clusters >= version 1.6.
+
+In Minikube, the default virtual machine driver's image doesn't contain the `sch_netem` kernel module in earlier versions. You can use `none` driver (if your host is Linux with the `sch_netem` kernel module loaded) to try these chaos actions using Minikube or [build an image with sch_netem by yourself](https://minikube.sigs.k8s.io/docs/contributing/iso/).
```
Comment (anotherrachel) [inline doc/get_started_on_minikube.md:58]: > or [build an image with sch_netem by yourself](https://minikube.sigs.k8s.io/docs/contributing/iso/)

This is a broken link.

Comment (yeya24) [reply doc/get_started_on_minikube.md:58]: Thanks for this! The link has been updated to `https://minikube.sigs.k8s.io/docs/contrib/building/iso/`. Could you please update it in this PR?

File: doc/sidecar_configmap.md [hunk 2]
```diff
@@ -69,35 +71,39 @@ data:
...
 * **volumeMounts**: defines the new volumeMounts or overwrite the old volumeMounts of the each containers in target pods.
 * **volume**: defines the new volumes for the target pod or overwrite the old volumes in target pods.
-* **postStart**: called after a container is created first. If the handler fails, the containers will failed. 
+* **postStart**: called after a container is created first. If the handler fails, the containers will failed.
 Key defines for the name of deployment container. Value defines for the Commands for stating container.
 
 ### Containers
 
-#### chaosfs
+#### `chaosfs`
```
Comment (yeya24) [inline doc/sidecar_configmap.md:90]: Do we need this?

Comment (anotherrachel) [reply doc/sidecar_configmap.md:90]: You mean the backtick(`)? I am not sure, actually. But the following paragraphs use like that.

File: doc/get_started_on_minikube.md (hunk 1 above)
Comment (anotherrachel) [reply doc/get_started_on_minikube.md:58]: Sure.

File: doc/sidecar_configmap.md (hunk 2 above)
Comment (yeya24) [reply doc/sidecar_configmap.md:90]: This seems odd in Markdown. I think it is better to remove the backticks. What do you think?

File: doc/dev_hello_world.md [hunk 3]
```diff
@@ -139,21 +137,22 @@ metadata:
 
 ## Make the docker image
```
Comment (yeya24) [inline doc/dev_hello_world.md:138]: Do we need to update to `Docker` in the title?

Comment (anotherrachel) [reply doc/dev_hello_world.md:138]: Good point. I also referred to their [docs](https://docs.docker.com/glossary/), and it turns out both "Docker" and "docker" are acceptable. So for this one I just let it be. :)

File: doc/sidecar_configmap.md (hunk 2 above)
Comment (anotherrachel) [reply doc/sidecar_configmap.md:90]: It's totally OK to use backticks in titles. The point is to use them consistently. 

We often use backticks to frame the following elements:

* Variable name
* Variable value
* Error message
* Configuration item name
* Field name
* The element that needs emphasis (for example, table name, column name, schema name, etc.)

**Comment by anotherrachel** [comment]
@cwen0 @zhouqiang-cl PTAL, thanks

File: doc/time_chaos.md [hunk 4]
```diff
@@ -41,6 +43,6 @@ Description:
 
 ## Limitation
 
-* Time modification will only be injected into the main process of container
-* Time chaos has no effect on pure syscall `clock_gettime`
-* All injected vdso call will use pure syscall to get realtime, so clock related function call will be much slower.
\ No newline at end of file
+* Time modification only can be injected into the main process of container.
```
Comment (dcalvin) [inline doc/time_chaos.md:46]:
```suggestion
* Time modification can only be injected into the main process of container.
```

File: doc/time_chaos.md [hunk 5]
```diff
@@ -41,6 +43,6 @@ Description:
...
+* Time chaos has no effect on pure syscall `clock_gettime`.
+* All injected vdso call use pure syscall to get realtime, so clock-related function call becomes much slower.
```
Comment (dcalvin) [inline doc/time_chaos.md:48]:
```suggestion
* All injected vDSO calls use pure system calls to get realtime, so clock-related function call can be much slower.
```

Comment (dcalvin) [reply doc/time_chaos.md:48]: Do we need to explain vDSO?  Also what do we mean by realtime?

**Pull request merged**